### ISSUE TITLE
[lf-laserfiche-lite.css] Styles are not specific enough

### ISSUE THREAD

**Issue by alexgomezlf** [Laserfiche/lf-ui-components#41]
Some styles in the style sheet are not descriptive enough and could affect users styles on accident.

For example,
```
#description {
  color: #0069CF;
  font: 600 14px "Open Sans";
}
#cancelButton {
  margin-left: 10px;
}
```

**Comment by alexgomezlf** [comment]
These also affect `lf-ms-office-lite.css`

**Comment by Ke-Jin-Laserfiche** [comment]
fixed by #43
